spec/models/program_spec.rb

```diff
@@ -256,7 +256,7 @@
     it 'returns an array with the types for which a cfp doesn\'t exist' do
       expect(program.remaining_cfp_types).to eq(Cfp::TYPES)
       create(:cfp, cfp_type: 'events', program: program, end_date: Date.current + 1)
-      expect(program.remaining_cfp_types).to eq([])
+      expect(program.remaining_cfp_types).to eq(["booths"])
```

Comment:
* There are `events` so the remaining type is `booths`
* There are `booths` so the remaining type is `events`
* There are both `events` and `booth` so the remaining type is empty
* Cfp is empty so the remaining types are both of them